src/encoded/static/components/genome_browser.js

```diff
@@ -445,7 +445,13 @@ class GenomeBrowser extends React.Component {
 
     handleOnFocus() {
         this.setState({ showAutoSuggest: false });
-        getCoordinateData(`${this.context.location_href.split('/experiments/')[0]}/suggest/?genome=${this.state.genome}&q=${this.state.searchTerm}`, this.context.fetch).then((response) => {
+        let coordinateHref = '';
```

Comment:
This is fine. But I think it can be simplified to be: 

```
const type = this.context.location_href.indexOf('/experiments') > -1 ? ‘experiment : ‘annotations’;

const coordinateHref = `${this.context.location_href.split(‘/${type}/‘)[0]}/suggest/?genome=${this.state.genome}&q=${this.state.searchTerm}`;
```

Reply:
I like this way of writing the code better, it is more succinct!